feat(search): Update docs to reflect new my_teams filter (#7257)
getsentry/sentry-docs: src/docs/product/sentry-basics/search/searchable-properties.mdx

| age | Returns issues created since the time defined by the value. The syntax is similar to the Unix find command. Supported suffixes: `m - minutes`, `h - hours`, `d - days`, `w - weeks`. For example, `age:-24h` returns isssues that are new in the last 24 hours, while `age:+12h` returns ones that are older than 12 hours. Entering `age:+12h age:-24h` would return issues created between 12 and 24 hours ago. |  | x |  | relative time |
| apdex(threshold) | Returns results with the [Apdex score](/product/performance/metrics/#apdex) that you entered. Values must be between `0` and `1`. Higher apdex values indicate higher user satisfaction. | x |  |  | number |
| app.in_foreground | Indicates if the app is in the foreground or background. Values are `1/0` or `true/false` | x | x |  | boolean |
| assigned | Returns issues assigned to the defined user(s) or team(s). Values can be a user ID (your email address), `me` for yourself, `me_or_none` for yourself or no assignee, `my_teams` for teams you belong to, or # followed by the team name, that is `#team-name`. |  | x |  | team or org user |

> | assigned_<br></br>or_suggested | Returns issues assigned to or suggested to be assigned to the defined user(s) or team(s). Suggested assignees are determined by matching [ownership rules](/product/issues/ownership-rules/) and [suspect commits](/product/issues/suspect-commits/). Values can be a user ID (your email address), `me` for yourself, `me_or_none` for yourself or no assignee/suggestions, `my_teams` for teams you belong to, or # followed by the team name, that is `#team-name`. |  | x |  | team or org user |

Reviewer: ```suggestion
| assigned_<br></br>or_suggested | Returns issues that are either assigned to or suggested as something that should be assigned to defined user(s) or team(s). Suggested assignees are found by matching [ownership rules](/product/issues/ownership-rules/) with [suspect commits](/product/issues/suspect-commits/). Values can be a user ID (your email address), `me` for yourself, `me_or_none` for yourself or no assignee/suggestion, `my_teams` or `#team-name` for teams you belong to. |  | x |  | team or org user |
```
